<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBoard</class>
 <widget class="QWidget" name="KeyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dot">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_b">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ensure">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_underline">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_capsmall">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CAPS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancle">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_backspace">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearall">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>clear_all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_q">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_e">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_r">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_i">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_y">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_o">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_u">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_p">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_t">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_w">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_j">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_a">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_d">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_g">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_l">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_k">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_s">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_f">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_h">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_v">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_m">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_n">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_z">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_x">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_c">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
